### PR TITLE
[feature](profile)  support simply  profile 

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes
A Simplified Version of the Profile

Divided into three levels:
Level 2: The original profile.
Level 1: Instances with identical structures are merged, utilizing concatenation for info strings, and recording the extremum for time types.

Note that currently, this is purely experimental, simplifying the profile on the frontend (you can view profiles at any level).

Subsequently, we will transition the simplification process to the backend. At that point, due to the simplification being done on the backend, viewing profiles at other levels won't be possible.

Due to the issue with the pipeline structure, the active time does not accurately reflect the time of the operators.

```
set enable_simply_profile = false;
set enable_simply_profile = true;
```

such as 

```
                    VAGGREGATION_NODE  (id=9):
                          -  AggInfos::  (_is_merge:  false,  _needs_finalize:  false,  Streaming  Preaggregation:  true,  agg  size:  0,  limit:  -1)
                          -  BuildTime:  [  avg:  3.526ms  ,  max:  4.748ms  ,  min:  2.68ms  ]
                          -  ExprTime:  [  avg:  5.210us  ,  max:  7.882us  ,  min:  2.869us  ]
                          -  HashTableComputeTime:  [  avg:  3.451ms  ,  max:  4.642ms  ,  min:  2.13ms  ]
                          -  HashTableEmplaceTime:  [  avg:  3.444ms  ,  max:  4.633ms  ,  min:  2.9ms  ]
                          -  HashTableInputCount:  100.0K  (100000)
                          -  HashTableSize:  560
                          -  RowsReturned:  560
                          -  RowsReturnedRate:  1.476915M  /sec
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.25 seconds
 stream load tsv:          538 seconds loaded 74807831229 Bytes, about 132 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17162046035 Bytes

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.11 seconds
 stream load tsv:          551 seconds loaded 74807831229 Bytes, about 129 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.6 seconds inserted 10000000 Rows, about 337K ops/s
 storage size: 17162063952 Bytes

**Comment by Mryange** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 47.02 seconds
 stream load tsv:          533 seconds loaded 74807831229 Bytes, about 133 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17162203198 Bytes

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p1

File: fe/fe-core/src/main/java/org/apache/doris/common/util/RuntimeProfile.java [hunk 1]
```diff
@@ -299,12 +304,220 @@ public void prettyPrint(StringBuilder builder, String prefix) {
...
+        }
+    }
+
+    private static void removeZeroeCounter(Set<String> childCounterSet, String childCounterName, Counter counter) {
+        if (counter.getValue() == 0) {
+            childCounterSet.remove(childCounterName);
+        }
+    }
+
+    // Will be used in the future.
```
Comment (HappenLee) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/RuntimeProfile.java:453]: remove the code of this

**Comment by Mryange** [comment]
run buildall

**Review by wm1581066 — APPROVED** [review]
LGTM

**Comment by Mryange** [comment]
run arm

**Review by HappenLee — DISMISSED** [review]
LGTM

**Review by Gabriel39 — DISMISSED** [review]
LGTM

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run clickbench

**Review by Gabriel39 — DISMISSED** [review]
LGTM

**Review by HappenLee — DISMISSED** [review]
LGTM

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Comment by Mryange** [comment]
run p0

**Review by zhangstar333 — APPROVED** [review]

**Comment by yiguolei** [comment]
Not  merge to 2.0.3, new profile will merge to 2.0 branch using a new pr

**Pull request merged**